### ISSUE TITLE
🐛 biome.lspBin didn't work after update to latest extension

### ISSUE THREAD

**Issue by SleepDevil** [biomejs/biome-vscode#161]
### VS Code version

1.87.0

### Extension version

2.2.1

### Biome version

1.5.3

### Operating system

- [ ] Windows
- [X] macOS
- [ ] Linux

### Description

I use settings biome.lspBin to find my biome,however, it doesn't work after today I update this extension.And I rollback to v2.1.2, it still works.
```
"biome.lspBin": "./common/autoinstallers/rush-format/node_modules/@biomejs/biome/bin/biome",
```
Maybe you should well test before publish.I'm tired of finding new bugs...
I will keep use old versions.

### Steps to reproduce

Use the same settings and will fail in the latest extension.

### Expected behavior

work as before.

### Does this issue occur when using the CLI directly?

No

**Comment by nhedger** [comment]
> Maybe you should well test before publish.I'm tired of finding new bugs...
I will keep use old versions.

Hi @SleepDevil,

Thanks for opening this issue. I understand your frustration with this bug and apologize for any inconvenience it has caused.

It's important to remember that our project is open source, and as such, it relies on contributions from the community, including testing and reporting bugs. While we strive for quality releases, it's not always possible to catch every issue before publishing. Your input is crucial in helping us identify and address these issues effectively.

However, I must emphasize that while feedback is encouraged, it's essential to maintain a respectful and constructive tone in our communication. Comments that include personal attacks or derogatory remarks are not in line with our community guidelines.

That being said, I've released [v2.2.2](https://github.com/biomejs/biome-vscode/releases/tag/v2.2.2), which should solve this issue. Please try it and let me know if it does.

**Comment by SleepDevil** [comment]
In fact, I mean no hostility, if you feel offended, I'd like to say sorry to you.
I just want to report a bug, and it's really frustrated when you open vscode and update this extension, and you find something went wrong. Especially when I've met too many bugs with the same extension.
Last, thanks for your work.
